_topic_map.yml

```diff
@@ -1345,7 +1345,7 @@ Topics:
     File: preparing-cluster-for-cnv
   - Name: Installing container-native virtualization
     File: installing-container-native-virtualization
-  - Name: Installing the `virtctl` client
+  - Name: Installing the virtctl client
```

Comment:
@ousleyp @aburdenthehand Backticks don't get replaced properly in the TOC and they're rendering, so I'm removing. Let me know if you have any concerns with that, thanks!

Reply:
Good catch, thanks!